src/sciline/visualize.py

```diff
@@ -21,6 +21,14 @@
 from .typing import Graph, Item, Key, get_optional
 
 
+class Digraph(Digraph):  # type: ignore[misc]
```

Comment:
You will have to import `graphviz.Digraph` differently, maybe
```suggestion
class Digraph(graphviz.Digraph):  # type: ignore[misc]
```

or something like that? As it stands, this may work, but looks very confusing.